languages/csharp/exercises/concept/properties/.meta/Example.cs

```diff
@@ -2,8 +2,8 @@
 
 public enum Units
 {
-    Kilograms
-    ,Pounds
+    Pounds
+    ,Kilograms
```

Comment:
```suggestion
    Pounds,
    Kilograms
```

See https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum